backend/protocol_rpc/transactions_parser.py

```diff
@@ -0,0 +1,78 @@
+# rpc/transaction_utils.py
+
+from typing import Any
+import rlp
+import codecs
+from eth_account._utils.legacy_transactions import Transaction, vrs_from
+from eth_account._utils.signing import hash_of_signed_transaction
+from eth_account import Account
+from eth_utils import to_checksum_address
+from hexbytes import HexBytes
+from rlp.sedes import binary
+
+
+def decode_signed_transaction(raw_tx):
```

Comment:
Is there a way to make this more readable? I personally find it hard to read all the `tx`, `txn` and `res`.

Reply:
done!